UWP/UWP.vcxproj

```diff
@@ -89,12 +89,14 @@
     <AppxAutoIncrementPackageRevision>True</AppxAutoIncrementPackageRevision>
     <AppxSymbolPackageEnabled>False</AppxSymbolPackageEnabled>
     <AppxBundle>Never</AppxBundle>
-    <PackageCertificateThumbprint>C8DEB388B9BC89D1DC61324E4E9D9FE6A796B7AA</PackageCertificateThumbprint>

```

Comment:
People keep changing this line, I haven't really involved myself in it.  Is this change intentional?

-[Unknown]

Reply:
No, I don't know actually why VS is changing this, since I never touched the certificate at all.